css/style.css

```diff
@@ -0,0 +1,307 @@
+:root {
+    /* colors */
+    --black: #000000;
+    --white: #ffffff;
+    --wine: #823468;
+    --pink-light: #F2B8CC;
+    --cream-light: #F6F4E9;
+    
+    /*  fw-fonts-weight */
+    --fw-400: 400;
+    --fw-500: 600;
+    --fw-700: 700;
+    --fw-900: 900;
+   
+    /* fs-font-size */
+    --fs-36px: 2.25rem;
+    --fs-24px: 1.5rem;
+    --fs-20px: 1.25rem;
+    --fs-12px: 0.75rem;
+
+}
+
+
+/* RESET */
+
+*,
+html {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    outline: 0;
+    border: 0;
+    
+    
+}
+
+
+/* Fonts */
+body{
+font-family: 'Maven Pro', sans-serif;
+
+    
+}
+
+h1{
+    color: var(--wine);
+    font-weight:var(--fw-900);
+    font-size:var(--fs-36px);
+}
+
+h2{
+    color: var(--white);
+    font-weight:var(--fw-900);
+    font-size:var(--fs-24px);
+}
+
+h3 {
+    color: var(--white);
+    font-weight:var(--fw-400);
+    font-size: var(--fs-20px);
+}
+
+p{
+    color: var(--white);
+    font-weight:var(--fw-400);
+    font-size: var(--fs-16px);
+}
+
+button{
+    color: var(--white);
+    font-weight:var(--fw-700);
+    font-size: var(--fs-12px);
+    background-color: var(--wine);
+    border-radius: 2rem;
+    width: 140px;
+    padding:4px 0.40rem;
+}
+
+
+
+
+.home {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-auto-rows: auto;
+    grid-template-areas:
+
+    
+    "h   h   h   n"
+    "w   w   w   w"
+    "hr  hr  hr  hr"
+    "pr  pr  pr  pr"
+    "ft  ft  ft  ft";
+
+}
+
+/* HEADER */
+.home-header {
+    display: flex;
+    align-items: center;
+    grid-area: h;
+    padding:0 1.625rem;
+    background-color:var(--white);
+    height: 100px;
+}
+
+/* NAV */
+.home-nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding:0 1.625rem;
+    align-items: center;
+    grid-area: n;
+}
+.home-nav-links{
+    display: none;
+}
+
+/* WELCOME  */
+.home-welcome{
+
+    padding: 1.625rem;
+    grid-area: w;  
+    background-color:var(--pink-light)
+}
+
+.home-welcome img{
```

Comment:
remember in class we discussed setting a height & width on the **parent** container for images, then setting either height **or** width to 100% on the img? This avoids affecting the aspect ratio of the image.